chapter11/rpi/Oogaboogoo/Time.pm

```diff
@@ -0,0 +1,34 @@
+package Oogaboogoo::Time;
+
+sub get_converters{
+
+  my %day = ( Sunday    => 'ark',
+              Monday    => 'dip',
+              Tuesday   => 'wap',
+              Wednesday => 'sen',
+              Thursday  => 'pop',
+              Friday    => 'sep',
+              Saturday  => 'kir',
+              );
+  my %month = ( January => 'diz',
+                February => 'pod',
+                March => 'bod',
+                April => 'rod',
+                May => 'sip',
+                June => 'wax',
+                July => 'lin',
+                August => 'sen',
+                September => 'kun',
+                October => 'fiz',
+                November => 'nap',
+                December => 'dep',
+                );
+
+  my $get_day = sub { return $day{ shift @_ } };
+
+  my $get_month = sub { return $month{ shift @_ } };
```

Comment:
Interesting use of closures, but what happens if someone supplies an invalid value? There's no documentation to explain!